resources/views/notifications/incident/new.blade.php

```diff
@@ -0,0 +1,17 @@
+@component('mail::message')
+# {{ trans('notifications.incident.new.mail.greeting', ['app_name' => Config::get('setting.app_name')]) }}
+
+{{ $incident->message }}
+
+@component('mail::button', ['url' => $actionUrl])
+{{ $actionText }}
+@endcomponent
+
+Thanks,<br>
+{{ config('app.name') }}
```

Comment:
This line is different with the greeting above: `Config::get('setting.app_name')`